generator.ts

```diff
@@ -135,9 +135,9 @@ export async function generate(token: string, configYaml: string): Promise<Repor
         }
       }
 
-      target.columnMap['Proposed'].push('Proposed', 'Not Started')
-      target.columnMap['In-Progress'].push('In-Progress', 'In progress', 'InProgress', 'Started')
-      target.columnMap['Accepted'].push('Accepted', 'Approved', 'Up Next')
+      target.columnMap['Proposed'].push('Proposed', 'New', 'Not Started', 'Ready for Triage')
```

Comment:
This is quick change to default in the column names guidance docs are asking for